### PR TITLE
Use /proc/meminfo for Memory module

### LINKED ISSUE #6: RAM free/used
screenshot_2018-08-09-22 07 44

@BlueGone You agree you have same problem on your computer ?

### PR DESCRIPTION
**Pull request by Robinhuett**
This parses `/proc/meminfo` as suggested in #6 

Closes #6

### CODE REVIEW

File: src/modules/memory.cpp [hunk 1]
```diff
@@ -13,13 +13,30 @@ waybar::modules::Memory::Memory(const Json::Value& config)
...
+  label_.set_tooltip_text(fmt::format("{:.{}f}Gb used", used_ram_gigabytes, 1));
+}
+
+void waybar::modules::Memory::parseMeminfo()
+{
+  int memtotal, memfree, memavail, membuffer, memcache;
+  FILE* info = fopen("/proc/meminfo","r");
+  if(fscanf (info, "MemTotal: %d kB MemFree: %d kB Buffers: %d kB Cached: %d kB",&memtotal, &memfree, &membuffer, &memcache) < 4) { // Old meminfo format 
+    fclose(info);
+    info = fopen("/proc/meminfo","r");
```
Comment (Alexays) [inline src/modules/memory.cpp]: Why close and re-open?

Comment (Robinhuett) [reply src/modules/memory.cpp]: You're right, after looking at it again, this can be replaced by `rewind(info);`

**Comment by Alexays** [comment]
There is no way to do that in modern cpp?
Or simply with ifstream :)

**Comment by Robinhuett** [comment]
The most cpp way, would be using `sysinfo`, which unfortunatly lacks `MemAvailable`.
But thanks for pointing me to `ifstream`, this looks a bit cleaner. I'll changed it later :)
And sorry, cpp is not my "native language"

**Pull request merged**